### PR TITLE
Update for CUDA 12.4.0

### PR DESCRIPTION
**Pull request by adibbley**
xref: https://github.com/conda-forge/cuda-feedstock/issues/17

### CODE REVIEW

**Comment by adibbley** [comment]
@conda-forge-admin , please re-render

**Comment by adibbley** [comment]
Travis seems to be stuck "Waiting to be queued" here, tried a restart with no luck yet.

**Comment by adibbley** [comment]
@conda-forge-admin, please restart ci

**Comment by jakirkham** [comment]
@conda-forge-admin , please re-render

**Comment by jakirkham** [comment]
@conda-forge-admin , please restart CI

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ requirements:
   run:
     - cuda-command-line-tools {{ version }}
     - cuda-visual-tools {{ version }}
-    - gds-tools 1.8.1.2                      # [linux64]
+    - gds-tools 1.9.0.20                      # [linux64]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Shouldn't we have this for `linux_aarch64` too?

```suggestion
    - gds-tools 1.9.0.20                      # [linux64 or aarch64]
```

Comment (adibbley) [reply recipe/meta.yaml]: Yes, thanks!

**Comment by jakirkham** [comment]
Travis CI failure was opaque. Running it offline the issue was clear

```
    Encountered problems while solving:
      - nothing provides requested cuda-command-line-tools 12.4.0.*
    
    Could not solve for environment specs
    The following package could not be installed
    └─ cuda-command-line-tools 12.4.0.*  does not exist (perhaps a typo or a missing channel).
```

`cuda-command-line-tools` packages are missing as the Travis CI job didn't run. Started one manually. More details in comment: https://github.com/conda-forge/cuda-command-line-tools-feedstock/pull/11#issuecomment-1986488228

**Comment by jakirkham** [comment]
@conda-forge-admin , please restart CI

**Comment by jakirkham** [comment]
Travis CI reached out stating they had deployed a fix and wanted us to test it

The automerge by the bot above serves as this test

Looking at the Travis CI requests can see where the automerge bot's push to `main` comes up. Note it states the build is created successfully

Screenshot 2024-03-08 at 4 11 36 PM

Also we can see the associated build that it created, which passed

Screenshot 2024-03-08 at 4 13 41 PM

So it looks like things are working again! 🎉

**Pull request merged**